api/v1alpha1/apparmorprofile_deepcopy.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
API files were moved under type-specific packages in https://github.com/kubernetes-sigs/security-profiles-operator/pull/178, so this will need to be moved and the Makefile updated